browser-extension/addTournesolRecommendations.js

```diff
@@ -170,17 +176,18 @@ const getTournesolComponent = () => {
       video_uploader.append(video.uploader);
       details_div.append(video_uploader);
 
-      // Tournesol score
-      // const video_score = document.createElement('p');
-      // video_score.className = 'video_text';
-      // video_score.append(
-      //   'Rated ' +
-      //     Number(video.tournesol_score).toFixed(0) +
-      //     ' points by ' +
-      //     video.rating_n_experts +
-      //     ' users',
-      // );
-      // details_div.append(video_score);
+       
+       const video_score = document.createElement('p');
+       video_score.className = 'video_text';
+       console.log(video)
+       video_score.append(
+         'Rated ' +
+           Number(video.rating_n_ratings).toFixed(0) +
+           ' points by ' +
```

Comment:
`rating_n_ratings` is the number of comparisons the video received. So a text like "X comparisons by N users" would be more appropriate.

And if we want to display the Tournesol score, it would need to be computed from the "criteria_scores".

Reply:
@lfaucon  do we prefer to report Tournesol score like 0.45 points by 23 contributors or 14 comparions by 23 contributors? number of comaprisons is understandable BUT
my suggestion is that reporting score would not be very helpful in numbers but then we can think about a descriptiove way like among the top ten percent of scores, rated 50% more than others, somethign lie that but needs backend analysis from my understaning. what do you think?

Reply:
1. I would prefer that we stick with "Rated X points by Y contributors". @sintaz1992 you use the wrong field in your implementation right now. You used the number of comparisons. If we choose to display "X comparisons by N users", then it is important that we implement a way to show the score too
2. About using percentiles, @sintaz1992  you may talk with @aidanjungo  about it on Discord, then make a proposal of how it would be computed , when , and how it would be displayed. if the team approves, we'll implement it.